src/components/oculus-touch-controls.js

```diff
@@ -0,0 +1,264 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var isControllerPresent = require('../utils/tracked-controls').isControllerPresent;
+
+var TOUCH_CONTROLLER_MODEL_BASE_URL = 'https://cdn.aframe.io/controllers/oculus/oculus-touch-controller-';
+var TOUCH_CONTROLLER_MODEL_OBJ_URL_L = TOUCH_CONTROLLER_MODEL_BASE_URL + 'left.obj';
+var TOUCH_CONTROLLER_MODEL_OBJ_MTL_L = TOUCH_CONTROLLER_MODEL_BASE_URL + 'left.mtl';
+var TOUCH_CONTROLLER_MODEL_OBJ_URL_R = TOUCH_CONTROLLER_MODEL_BASE_URL + 'right.obj';
+var TOUCH_CONTROLLER_MODEL_OBJ_MTL_R = TOUCH_CONTROLLER_MODEL_BASE_URL + 'right.mtl';
+
+var GAMEPAD_ID_PREFIX = 'Oculus Touch';
+
+var PIVOT_OFFSET = {x: 0, y: -0.015, z: 0.04};
+
+// currently, browser bugs prevent capacitive touch events from firing on trigger and grip;
```

Comment:
you would have to ask @tojiro -- looks like there is some small nonzero analog value always, that prevents not being touched